src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -2026,12 +2022,13 @@ public static MemtablePool getMemtableAllocatorPool()
     {
         long heapLimit = ((long) conf.memtable_heap_space_in_mb) << 20;
         long offHeapLimit = ((long) conf.memtable_offheap_space_in_mb) << 20;
+        final MemtableCleaner cleaner = ColumnFamilyStore::flushLargestColumnFamily;
         switch (conf.memtable_allocation_type)
         {
             case unslabbed_heap_buffers:
-                return new HeapPool(heapLimit, conf.memtable_cleanup_threshold, new ColumnFamilyStore.FlushLargestColumnFamily());
+                return new HeapPool(heapLimit, conf.memtable_cleanup_threshold, cleaner);
             case heap_buffers:
-                return new SlabPool(heapLimit, 0, conf.memtable_cleanup_threshold, new ColumnFamilyStore.FlushLargestColumnFamily());
+                return new SlabPool(heapLimit, 0, conf.memtable_cleanup_threshold, cleaner);
```

Comment:
@ekaterinadimitrova2 Outside of these two, the changes in DD are purely cosmetic, correct?